.github/workflows/pnp-input-api-build.yml

```diff
@@ -316,12 +316,21 @@ jobs:
               "apikey": "${{ env.NUGET_API_KEY }}"
             }]
 
+      - name: Get Identity token
+        id: identitytoken
+        uses: extenda/actions/identity-token@v0
+        with:
+          service-account-key: ${{ secrets.GCLOUD_AUTH_STAGING }}
+          service-account: braveheart-quotes-webclient@quotes-staging-ccdf.iam.gserviceaccount.com
```

Comment:
Not sure we can use this test service account email here

Reply:
Ok, which we can use instead? Or whom we should ask about it? Platform team?